src/Components/CommentsWidget.js

```diff
@@ -83,18 +77,50 @@ export class Comment extends preact.Component {
                     []
                 )}
                 <p>
-                    {this.props.message.split('\n').map((item, key) => {
-                        return (
-                            <span key={key}>
-                                {item}
-                                <br />
-                            </span>
-                        );
-                    })}
+                    {this.props.message
+                        .split('\n')
+                        .map(this.processLine)
+                        .flat()}
                 </p>
             </div>
         );
     }
+
+    /**
+     * Handle some very basic markup in comments, similar to what the generator allows.
+     *
+     * The following tags are supported:
+     *   - <bold>This text will be bold.</bold>
+     *   - <italic>This text will be italic.</italic>
+     *   - <link url="https://example.org">This text will link to example.org.</link> Note that the URL can technically
+     *       be missing, it will then produce a link without href. Also note that URLs are not checked at all.
+     *
+     * @param {string} line The line of text to process.
+     * @returns {Array} An array of the elements to insert instead of the line.
+     */
+    processLine = line => {
+        const processChunk = function(chunk) {
+            let chunks = [];
+            let remaining = chunk;
+            let match;
+
+            while (
+                (match = remaining.match(/<(?<tag>bold|italic|link)(?: url="(?<url>https:\/\/.+?)")?>(?<content>.+?)<\/\1>/))
```

Comment:
```suggestion
                (match = remaining.match(/<(?<tag>bold|italic|link)(?: url="(?<url>https?:\/\/.+?)")?>(?<content>.+?)<\/\1>/))
```

Much as I would like the web to be HTTPS-only, I'm pretty sure we'll want to link to some HTTP content at some point…